1.1.12 pip package fails to install on python 3.8.x

while borg 1.1.11 works on py38, 1.1.12 does not.

seems to be related to the bundled msgpack code (which did not change between 1.1.11 and 1.1.12, so maybe it is a cython code generation issue due to a different cython version):

```
    running build_ext
    building 'borg.algorithms.msgpack._packer' extension
    creating build/temp.linux-x86_64-3.8
    creating build/temp.linux-x86_64-3.8/src
    creating build/temp.linux-x86_64-3.8/src/borg
    creating build/temp.linux-x86_64-3.8/src/borg/algorithms
    creating build/temp.linux-x86_64-3.8/src/borg/algorithms/msgpack
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -D__LITTLE_ENDIAN__=1 -I/usr/include -I/usr/include -Isrc/borg/algorithms/zstd/lib -Isrc/borg/algorithms/zstd/lib/common -Isrc/borg/algorithms/zstd/lib/compress -Isrc/borg/algorithms/zstd/lib/decompress -Isrc/borg/algorithms/zstd/lib/dictBuilder -I/usr/include -I/home/user/w/aa/env/include -I/home/user/.pyenv/versions/3.8.3/include/python3.8 -c src/borg/algorithms/msgpack/_packer.cpp -o build/temp.linux-x86_64-3.8/src/borg/algorithms/msgpack/_packer.o
    src/borg/algorithms/msgpack/_packer.cpp: In function ‘PyCodeObject* __Pyx_CreateCodeObjectForTraceback(const char*, int, int, const char*)’:
    src/borg/algorithms/msgpack/_packer.cpp:332:86: error: invalid conversion from ‘PyObject* {aka _object*}’ to ‘int’ [-fpermissive]
               PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
                                                                                          ^
    src/borg/algorithms/msgpack/_packer.cpp:8310:15: note: in expansion of macro ‘__Pyx_PyCode_New’
         py_code = __Pyx_PyCode_New(
                   ^~~~~~~~~~~~~~~~
    src/borg/algorithms/msgpack/_packer.cpp:332:86: error: invalid conversion from ‘int’ to ‘PyObject* {aka _object*}’ [-fpermissive]
               PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
                                                                                          ^
    src/borg/algorithms/msgpack/_packer.cpp:8310:15: note: in expansion of macro ‘__Pyx_PyCode_New’
         py_code = __Pyx_PyCode_New(
                   ^~~~~~~~~~~~~~~~
    src/borg/algorithms/msgpack/_packer.cpp:332:86: error: too many arguments to function ‘PyCodeObject* PyCode_New(int, int, int, int, int, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*, int, PyObject*)’
               PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
                                                                                          ^
    src/borg/algorithms/msgpack/_packer.cpp:8310:15: note: in expansion of macro ‘__Pyx_PyCode_New’
         py_code = __Pyx_PyCode_New(
                   ^~~~~~~~~~~~~~~~
    In file included from /home/user/.pyenv/versions/3.8.3/include/python3.8/compile.h:5:0,
                     from /home/user/.pyenv/versions/3.8.3/include/python3.8/Python.h:138,
                     from src/borg/algorithms/msgpack/_packer.cpp:4:
    /home/user/.pyenv/versions/3.8.3/include/python3.8/code.h:122:28: note: declared here
     PyAPI_FUNC(PyCodeObject *) PyCode_New(
                                ^~~~~~~~~~
    error: command 'gcc' failed with exit status 1

```

--- a/docs/changes.rst
+++ b/docs/changes.rst
@@ -217,7 +217,7 @@ The best check that everything is ok is to run a dry-run extraction::
 Changelog
 =========
 
-Version 1.1.12 (2020-06-06)
+Version 1.1.13 (2020-06-06)
 ---------------------------
 
 Compatibility notes:
@@ -240,6 +240,14 @@ Compatibility notes:
   If WSL still has a problem with sync_file_range, you need to set
   BORG_WORKAROUNDS=basesyncfile in the borg process environment to
   work around the WSL issue.
+
+Fixes:
+
+- rebuilt using a current Cython version, compatible with python 3.8, #5214
+
+
+Version 1.1.12 (2020-06-06)
+---------------------------
 
 Fixes:
 

